sdformat_mjcf/sdformat_mjcf/converters/geometry.py

```diff
@@ -74,7 +76,30 @@ def add_geometry(body, name, pose, sdf_geom):
         geom.type = "sphere"
         geom.size = [sphere_shape.radius()]
     elif sdf_geom.mesh_shape():
-        raise RuntimeError("Meshes are not yet supported")
+        mesh_shape = sdf_geom.mesh_shape()
+        uri = mesh_shape.uri()
+        dirname = os.path.dirname(mesh_shape.file_path())
+        extension_tokens = os.path.basename(mesh_shape.uri()).split(".")
+        if (len(extension_tokens) > 1):
+            extension = extension_tokens[1]
+        else:
+            raise RuntimeError("Unable to find the mesh extension {}"
+                               .format(uri))
+        file_without_extension = os.path.splitext(
+            os.path.basename(mesh_shape.uri()))[0]
+        if 'http://' in uri or 'https://' in uri:
+            raise RuntimeError("Fuel meshes are not yet supported")
+        geom.type = "mesh"
+        if "obj" in extension or "stl" in extension:
```

Comment:
Since pymjcf itself checks if the mesh file format is supported, we should leave it to the library. This will also ensure that additional file formats can be supported by pymjcf in the future without changing this list here.

https://github.com/deepmind/dm_control/blob/d72c22f3bb89178bff38728957daf62965632c2f/dm_control/mjcf/attribute.py#L520